edb/server/protocol/auth/__init__.py

```diff
@@ -27,7 +27,7 @@
 from . import scram
 
 
-async def handle_request(request, response, path_parts, server):
+async def handle_request(request, response, path_parts, server, tenant):
```

Comment:
Changed to pass in `tenant` only in a future commit: 0b720aac7e8a0c032d3a0b327aa8c1556981b2e5